components/micro-gateway-cli/src/main/resources/templates/jwtRevocation.mustache

```diff
@@ -2,7 +2,7 @@ map<string> receivedRevokedTokenMap = gateway:getRevokedTokenMap();
     boolean jmsListenerStarted = gateway:initiateTokenRevocationJmsListener();
 
     boolean useDefault = gateway:getConfigBooleanValue(gateway:PERSISTENT_MESSAGE_INSTANCE_ID,
-    gateway:PERSISTENT_USE_DEFAULT, false);
+        gateway:PERSISTENT_USE_DEFAULT, gateway:DEFAULT_PERSISTENT_USE_DEFAULT);
```

Comment:
indentation issue

Reply:
I believe the indentation is correct here. Kindly insert the suggestion<cmd-g> if not.